### DISCUSSION TITLE
Teams webhook problem - anybody can help please?

### DISCUSSION THREAD

**Discussion by klkpl** [snipe/snipe-it#14306] [Q&A] [answered]
Hey, I configured webhook for teams and used in Settings/Integration. 
When I press test button, it works, I get notification in Teams channel.
However, when I deploy asset to the user, no notification is sent. During asset assignment I have that blue bar saying webhook will post to the teams, etc. Nothing happens.
Am I doing something wrong? It's a massive feature, so helpful, would love to make this work, what am I missing? Thanks...

**Comment by snipe** [comment]
Try updating to latest on master. I know a few webhook things were fixed recently.

**Reply by klkpl** [reply]
Thanks for a quick response I appreciate it. I think I am on the latest master already though, only did php upgrade.php today :-) 
My version is: Version v6.3.1 - build 12672 (master) 
Thanks nevertheless.

**Comment by Godmartinz** [comment]
> Hey, I configured webhook for teams and used in Settings/Integration. When I press test button, it works, I get notification in Teams channel. However, when I deploy asset to the user, no notification is sent. During asset assignment I have that blue bar saying webhook will post to the teams, etc. Nothing happens. Am I doing something wrong? It's a massive feature, so helpful, would love to make this work, what am I missing? Thanks...

@klkpl sorry to hear this isn't working for you. I am having some trouble reproducing on my end. Is there anything that pops up in your laravel.log when you checkin/checkout?

**Reply by klkpl** [reply]
Hey, cheers for this. I checked this log and I see this:

production.WARNING: Exception caught during checkout notification: Driver [NotificationChannels\MicrosoftTeams\MicrosoftTeamsChannel] not supported.

Thank you for any ideas :-)

**Reply by klkpl** [reply]
@Godmartinz but if I go to settings and test the webhook it works fine every single time. I even recreated the webhook in Teams, but that didn't change anything. Thanks!

**Reply by marcusmoore** [reply, marked as answer]
@klkpl can you run `composer install`? It should have run inside of `upgrade.php` but it seems like the package that actually handles sending notifications, `laravel-notification-channels/microsoft-teams`, might not be installed on your system?

(That package handles notification sending but the testing of the webhook goes through a different flow)

**Reply by klkpl** [reply]
Thank you, thank you and one more time thank you :-) Composer install fixed this for me :-) It works now!! You are brilliant guys. All the best.
